addon/globalPlugins/remoteClient/local_machine.py

```diff
@@ -77,4 +79,13 @@ def set_clipboard_text(self, text, **kwargs):
 		api.copyToClip(text=text)
 
 	def send_SAS(self, **kwargs):
-		ctypes.windll.sas.SendSAS(0)
+		"""
+		This function simulates as "a secure attention sequence" such as CTRL+ALT+DEL.
+		SendSAS requires UI Access, so we provide a warning when this fails.
+		This warning will only be read by the remote NVDA if it is currently connected to the machine.
+		"""
+		if systemUtils.hasUiAccess():
+			ctypes.windll.sas.SendSAS(0)
+		else:
+			ui.message("No permission on remote device to trigger CTRL+ALT+DEL")

```

Comment:
Please make this message translatable by wrapping the string in _()

Reply:
```suggestion
			ui.message(_("No permission on remote device to trigger CTRL+ALT+DEL"))
```